settings.gradle.kts

```diff
@@ -5,12 +5,13 @@
 rootProject.name = "kotlinx-knit"
 include("kotlinx-knit-test")
 
-val kotlinVersion: String by settings
-val dokkaVersion: String by settings
-val pluginPublishVersion: String by settings
 
 pluginManagement {
     resolutionStrategy {
+        val kotlinVersion: String by settings
+        val dokkaVersion: String by settings
+        val pluginPublishVersion: String by settings
```

Comment:
Just question: is this move of props to a local block just a matter of style or something functional?

Reply:
This change is forced by Gradle update. It _seems_ that Gradle now compiles and executes `pluginManagement` block before everything else and separately from everything else. Everything declared outside of `pluginManagement` and accessed from within `pluginManagement` triggeres unresolved reference and script compilation error